server/deviceflowhandlers.go

```diff
@@ -151,7 +151,10 @@ func (s *Server) handleDeviceCode(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-func (s *Server) handleDeviceToken(w http.ResponseWriter, r *http.Request) {
+func (s *Server) handleDeviceTokenGrant(w http.ResponseWriter, r *http.Request) {
```

Comment:
I think this name is a bit confusing and ambiguous with the `handleDeviceToken` name. Maybe call it something like `deprecatedDeviceTokenHandler` or `handleDeviceTokenDeprecated`.